### DISCUSSION TITLE
unable to connect to the mongodb server in catalogue

### DISCUSSION THREAD

**Discussion by SindhuGutha09** [learndevopsonline/learndevopsonline#1253] [Q&A] [answered]
Hi @r-devops,
I have tried multiple ways to connect to mongodb serever in the catalogue but it's says there is a connection timeout and the connection failed.
I have taken one Domain name server which is sgdevrobo.online and I have updated in each and every server with that name in the configuration file. 
PFB screenshots for the reference of the connections which I did so far 
![Screenshot (1)](https://github.com/learndevopsonline/learndevopsonline/assets/138164168/9925c64b-4cc6-402f-a844-e57c03dab0f4)
![Screenshot (4)](https://github.com/learndevopsonline/learndevopsonline/assets/138164168/4c0cca65-7d58-407a-9990-37b0fbf51e54)

![Screenshot (2)](https://github.com/learndevopsonline/learndevopsonline/assets/138164168/f0f07d8f-3bb4-42d9-82d2-b110c3ee127b)

 I have checked all the possible ways but am unable to make a connection to mongodb can you please help me this.
regards,
sindhuja gutha.

**Comment by maheshbusireddy** [comment]
@ChowdarySindhu servers are not reachable to check the issue.

**Reply by SindhuGutha09** [reply]
@maheshbusireddy I have started my servers now can please check it again am facing the same issue.

**Reply by SindhuGutha09** [reply]
![Screenshot (5)](https://github.com/learndevopsonline/learndevopsonline/assets/138164168/41154ef2-664d-4286-a794-6c7ce40483a5)

**Reply by maheshbusireddy** [reply, marked as answer]
@ChowdarySindhu you need to update the DNS record after restart of servers with the new ip address
Here showing clearly the error could  not connect to  mongodb.sgdevrobo.online that mean you should update the dns record every time when restart the servers.
![image](https://github.com/learndevopsonline/learndevopsonline/assets/99891108/4af81ce7-0571-4faf-8d05-4a382b0d8c09)

**Reply by SindhuGutha09** [reply]
resolved thank you
